src/forms/css/objects/_honeycomb.forms.objects.marketo.scss

```diff
@@ -35,6 +35,7 @@ form[id^="mktoForm_"] {
 
 .mktoLabel {
     display: flex;
+    font-weight: 500;
```

Comment:
Should this be applied to all labels, rather than just Marketo form labels?

Reply:
Good question - it probably does make most sense to have this on all labels.
I put it here thinking it's a style update mainly for long forms and short forms would still keep bold labelling, but thinking again it's better to have a consistent weight everywhere, and if we really want to make any ad hoc labels bold afterward we still can. So I'll update that!